resources/styles/global/forms.less

```diff
@@ -1,3 +1,5 @@
+@form-error-color: #E35C3B;
```

Comment:
Hm, I believe that's the only time we use that color anywhere, but I can put it into variables.less.  I was under the impression that for colors that are only used in one file we would want to keep it locally defined.